maintenance_mode/http.py

```diff
@@ -1,15 +1,83 @@
 # -*- coding: utf-8 -*-
 
 """HTTP response utilities."""
+import sys
+import re
+
 import django
 from django.core.exceptions import ImproperlyConfigured
-from django.shortcuts import render, redirect
+from django.core.urlresolvers import NoReverseMatch, resolve, reverse
+from django.shortcuts import redirect, render
 from django.template import RequestContext
 from django.utils.cache import add_never_cache_headers
 from django.utils.module_loading import import_string
 
 from maintenance_mode import settings
```

Comment:
`from maintenance_mode import settings, utils`

Reply:
That's a very uncommon import style. Is there any reason for that?

Reply:
Why uncommon import style? We are using all the methods of the modules, so there is no difference.

Reply:
I mean uncommon among python projects. Also imports from `maintenance_mode` often import whole modules, while the imports from Django import only functions or classes. I'd like to follow the style rules, but I don't see any.

Reply:
Can we somehow sort this out as well?